### PR TITLE
libswoc: Replace IpMap in ControlMatcher.

### PR DESCRIPTION
**Pull request by SolidWallOfCode**

### CODE REVIEW

File: proxy/ControlMatcher.cc [hunk 1]
```diff
@@ -651,23 +651,21 @@ template <class Data, class MatchResult>
...
 void
 IpMatcher<Data, MatchResult>::Print() const
 {
-  printf("\tIp Matcher with %d elements, %zu ranges.\n", num_el, ip_map.count());
-  for (auto &spot : ip_map) {
-    char b1[INET6_ADDRSTRLEN], b2[INET6_ADDRSTRLEN];
-    printf("\tRange %s - %s ", ats_ip_ntop(spot.min(), b1, sizeof b1), ats_ip_ntop(spot.max(), b2, sizeof b2));
-    static_cast<Data *>(spot.data())->Print();
+  std::string tmp;
+  printf("\tIp Matcher with %d elements, %zu ranges.\n", num_el, ip_addrs.count());
```
Comment (zwoop) [inline proxy/ControlMatcher.cc:665]: This looks like debugging remnant?

**Review by zwoop — CHANGES_REQUESTED** [review]
Can we not remove tscore/IpMap.h yet? There will be more changes needed to nuke it?

File: proxy/ControlMatcher.cc (hunk 1 above)
Comment (SolidWallOfCode) [reply proxy/ControlMatcher.cc:665]: It's part of the `Print` method which is used from a higher layer.

**Comment by SolidWallOfCode** [comment]
No, we can't remove `IpMap` yet. I tried that and it ended up being a very large PR. As I'm constantly told to do, I'm breaking it down into specific uses of `IpMap` and fixing those.

**Review by zwoop — APPROVED** [review]

**Pull request merged**